File: app/src/main/java/com/example/mobv2/ui/fragments/comments/CommentsFragmentViewModel.java
package com.example.mobv2.ui.fragments.comments;

import androidx.lifecycle.ViewModel;

import com.example.mobv2.models.Post;

public class CommentsFragmentViewModel extends ViewModel
{
    private Post post;

    public Post getPost()
    {
        return post;
    }

    public void setPost(Post post)
    {
        this.post = post;
    }
}

File: app/src/main/java/com/example/mobv2/ui/fragments/comments/CommentsFragmentViewModel.kt
package com.example.mobv2.ui.fragments.comments

import androidx.databinding.Bindable
import com.example.mobv2.BR
import com.example.mobv2.utils.ObservableViewModel
import com.example.mobv2.adapters.PostsAdapter.PostItem

class CommentsFragmentViewModel : ObservableViewModel() {
    var postItem: PostItem? = null
}